src/GraphWebhooks/Controllers/HomeController.cs

```diff
@@ -2,23 +2,38 @@
 // Licensed under the MIT license.
 
 using System.Diagnostics;
+using GraphWebhooks.Models;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
-using GraphWebhooks.Models;
 
 namespace GraphWebhooks.Controllers;
 
+/// <summary>
+/// The controller for the home page.
+/// </summary>
 public class HomeController : Controller
 {
+    /// <summary>
+    /// Initializes a new instance of the <see cref="HomeController"/> class.
+    /// </summary>
+    public HomeController()
```

Comment:
this could have been a primary constructor as well for consistency

Reply:
Good catch